### DISCUSSION TITLE
SNES/SFC cable adapter question

### DISCUSSION THREAD

**Discussion by Sidecar940** [darthcloud/BlueRetro#117] [Q&A] [answered]
Hello, I am building the universal adapter with the DB25, and I have a question with the SNES / SFC cable adapter, if I use the pcb level shifter do I also have to add the components of the DIY Through-hole part?

**Comment by darthcloud** [comment, marked as answer]
No, you use the pcb section part list.

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#smd-cable-pcb-5
